lab-kyle/lib/note-crud.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
```

Comment:
Things look good for the most part. Found some code that can be cleaned up a bit.